app.py

```diff
@@ -117,6 +123,11 @@ def _do_sync(
             int(only_dependency_monkey_documents),
         )
     )
+    environment_varibale = "THOTH_NAMESPACE"
+    if os.getenv(environment_varibale):
+        namespace = os.getenv(environment_varibale)
```

Comment:
```python
environment_variable = os.getenv("THOTH_NAMESPACE")
if not environment_variable:
        _LOGGER.warning("Namespace variable not provided for %r", environment_varibale)
```

Note the `%r` to expand the warning message.

Reply:
Thanks!